Breeding-Insight/bi-web
# BI-1195 - 4.3 Exp Preview: Append Experiment with Observations

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1195

Updated the display of the experiment preview table as some of the data point names changed during the backend implementation of BI-1195



# Dependencies
_Please include any dependencies to other code branches, testing configurations, scripts to be run, etc._

# Testing
1. Upload a new experiment file
2. Verify that all data points show up on the preview table


# Checklist:
